generator/generator_tests/relation_tags_tests.cpp

```diff
@@ -0,0 +1,263 @@
+#include "testing/testing.hpp"
+
+#include "generator/generator_tests/common.hpp"
+#include "generator/relation_tags.hpp"
+#include "generator/intermediate_data.hpp"
+
+namespace relation_tags_tests
+{
+using namespace generator_tests;
+using namespace generator;
+using namespace feature;
+using namespace std;
+
+// In memory relations storage (copy-n-paste from collector_building_parts_tests.cpp).
+class TestOSMElementCacheReader : public generator::cache::OSMElementCacheReaderInterface
+{
+public:
+  TestOSMElementCacheReader(std::unordered_map<generator::cache::Key, RelationElement> & m)
+    : m_mapping(m)
+  {
+  }
+
+  // OSMElementCacheReaderInterface overrides:
+  bool Read(generator::cache::Key /* id */, WayElement & /* value */) override { UNREACHABLE(); }
```

Comment:
What is the meaning of UNREACHABLE here? It it should not be called, please use `CHECK(false, ("Should not be called"));`

Reply:
Replaced with `CHECK(false, ...)`.
Also replaced in `collector_building_parts_tests.cpp`.